.github/workflows/tests.yml

```diff
@@ -75,8 +75,10 @@ jobs:
           node-version: ${{ matrix.node }}
       - name: Monorepo install
         uses: ./.github/actions/yarn-nm-install
+      - name: Monorepo build
+        uses: ./.github/actions/run-build
       - name: Run docs build
-        run: cd docs && yarn build
```

Comment:
This is needed to get the CI to pass here, it's a bug with the docs_build job I added last week